src/index.ts

```diff
@@ -1 +1 @@
-export * from './spacetimedb.js';
+export { SpacetimeDBClient } from "./spacetimedb";
```

Comment:
This will break anything that needs something other than `SpacetimeDBClient` and we also import other stuff in generated types (like `AlgebraicType` etc).

Reply:
Fixed this